maxim/logger/components/generation.py

```
@@ -198,8 +204,12 @@ def __init__(
             if self.provider not in valid_providers:
                 self.provider = "unknown"
         else:
-            self.provider = "unknown"
+            self.provider = "unknown"        
         self.messages.extend([m for m in (final_config.get("messages") or [])])
```

Ruff (0.11.9)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)